commit-message-hook.sh

```diff
@@ -0,0 +1,22 @@
+#!/bin/bash
+# Include any branches for which you wish to disable this script
+if [ -z "$BRANCHES_TO_SKIP" ]; then
```

Comment:
Tested using this on master, this part doesn't seem to work as intended as it complains that master is not following the right naming conventions.